d3-scripts/src/d3_scripts/resolve_behaviour_rules.py

```diff
@@ -0,0 +1,33 @@
+import networkx as nx
+from .check_behaviours_resolve import BehaviourMap
```

Comment:
If using `BehaviourJson` in this file as a type.
```suggestion
from .check_behaviours_resolve import BehaviourMap, BehaviourJson
```